src/articles/2020/cdk-tips-01-how-to-use-local-cdk-commands.md

```diff
@@ -176,7 +176,8 @@ which ships with all modern distributions of NodeJS.
 So, instead of invoking `cdk init --language=<lang>`
 you would call `npx cdk init --language`,
 and `npx` will download the latest version of the `aws-cdk` package,
-and store it in a temporary directory.
+and store it in a temporary directory. You can use a specific `aws-cdk` package version if needed:
+`npx cdk@1.44.0 init --language`.
```

Comment:
Let's re-phrase this a little bit, considering the previous sentence mentions the latest version of the CDK already:

```suggestion
and `npx` will download the latest version of the `aws-cdk` package,
and store it in a temporary directory
(if you want to use a specific version of the CDK, instead of the latest,
you can do it by adding a version specifier to the command,
like `npx cdk@1.98.0 init --language=<lang>`).

Reply:
Done, thanks!